src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,57 @@
+import { FC } from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/types';
+
+type Props = {
+  todo: Todo;
+  activeIds: number[];
+  handleRemoveTodo: (buttonId: number) => void;
+};
+
+export const TodoItem: FC<Props> = ({
+  todo,
+  activeIds,
+  handleRemoveTodo,
+}) => {
+  const { title, completed, id } = todo;
+
+  return (
+    <div
+      className={cn(
+        'todo',
+        'item-enter-done',
+        { completed },
+      )}
+      data-cy="todo"
+    >
+      <div className="todo__status-label">
+        <input type="checkbox" className="todo__status" />
```

Comment:
Add a checked property to the input to set the initial state of the checkbox based on the 'completed' property of the todo.